_posts/ja/newsletters/2020-02-26-newsletter-ja.md

```diff
@@ -0,0 +1,149 @@
+
+    straggler problemを修正する一般的なアプローチは、冗長性を導入することです。上記の例では、支払者が0.01BTCの7回の部分支払いを行い、受信者が正常にルーティングされた支払いの最初の5回を請求することになります。問題は、受信者が7つすべてを要求して0.02BTCの過剰支払いを引き起こすのを防ぐ方法です。
+
+    Neuなどは_boomerang_コントラクトと呼ばれる新しいスキームを提示します。受信者は、[公的に検証可能な秘密の共有][pvss]スキームのシェアとして、支払い部分のプレイメージを選択します。上記の例では、7つの支払い前画像のうち6つから秘密を再構築できます。支払者は7つの支払部分(paymentparts)を作成しますが、支払者が完全な秘密を知っている場合、支払者は全額を返済する逆（ブーメラン）条件に関連付けられます。受取人が5つ以下の支払い部分を要求する場合、支払者は完全な秘密を知ることはできず、ブーメラン条項を呼び出すことはできません。また受取人が6つ以上の部分をチートして要求する場合、支払者は契約のブーメラン条項、および支払い部分のいずれも受取人が引き出すことはできません。
+
+    この論文では、シュノア署名スキームに基づいて、[アダプター署名][adaptor signatures]を使用したビットコインでのブーメラン契約の実装について説明します。Neuはまた、ECDSAを介してアダプター署名を作成することが可能であるため、現在のブーメラン契約は理論的にはBitcoinで実装できると述べています。
+
+- **匿名トランザクションに対するリモートサイドチャネル攻撃:** Florian Tramerは、Dan BonehおよびKenneth G. Patersonと、MoneroおよびZcashでのユーザープライバシーに対するサイドチャネルおよびトラフィック分析攻撃のタイミングに関する研究を発表しました（[transcript][side-channel txt]、[video][side-channel vid]、[paper][side-channel paper]）。
+
+    MoneroおよびZcashは、暗号技術（Moneroの[リング署名][ring signatures]および[bulletproofs][]およびZcashの[zk-SNARKs][]）を使用して送信者のID、受信者のID、および金額を隠すプライバシー重視の暗号通貨です。Tramerなどは、 これらの暗号構造が正しい場合でも、実装の詳細により、アイデンティティと量に関する情報がネットワーク上の悪意のある者に漏洩する可能性があることを示します。
+
+    MoneroまたはZcashノードがピアツーピアネットワークからトランザクションを受信すると、そのトランザクションはノードのウォレットに渡され、トランザクションがウォレットに属しているかどうかを判断します。トランザクションがウォレットに属する場合、ウォレットはトランザクションからデータと金額を復号化するために追加の計算を行う必要があり、ウォレットがこの追加の計算作業を行っている間にノードのピアツーピアアクティビティを一時停止すると、攻撃者は[タイミング攻撃][timing attack]を使用して、どのトランザクションがどのノードに関連付けられているかを発見できます。著者は、これらのタイミング攻撃がリモートで（ロンドンからチューリッヒへのWAN接続を介して）実行可能であり、同様のタイミング攻撃を使用してZcashトランザクションの金額を明らかにすることもできることを実証しています。
+
+    論文の攻撃はビットコインコアには適用されません。ビットコインコア・ウォレットが独自のトランザクションと他のトランザクションで行う計算の違いは最小限であり（高度な暗号化は含まれません）、v0.16以降、ウォレット操作はピアツーピアの振る舞いから非同期に処理されます（[Bitcoin Core #10286][]参照）。ただし、この論文の観察結果は、Bitcoinでシステムを実装する人にとって興味深い一般的な問題と言えるでしょう。つまり、ウォレットまたはアプリケーションの処理がピアツーピアの動作に影響を与えると、情報が漏洩する可能性があります。
+
+関連：Optechニュースレターは、昨年のStanford Blockchain Conferenceの講演の抜粋を[Newsletter#32][news46 sbc]にまとめています。
+
+## Bitcoin StackExchangeから選ばれたQ＆A
+
+*[Bitcoin StackExchange] [bitcoin.se]はOptech Contributor達が疑問に対して答えを探しに（もしくは他のユーザーの質問に答える時間がある場合に）アクセスする、数少ない情報ソースです。この月刊セクションでは、前回アップデート以降にされた、最も票を集めた質問・答えについて共有しています。*
+
+{% comment %}<!-- https://bitcoin.stackexchange.com/search?tab=votes&q=created%3a1m..%20is%3aanswer -->{%
+endcomment %}
+{% assign bse = "https://bitcoin.stackexchange.com/a/" %}
+
+- [SLIP39とBIP39の関係は？]({{bse}}93413) ユーザーThalis K.は、ニーモニックの生成と決定論的ウォレットシードへの変換の仕様であるBIP39（ビットコイン改善提案）とSLIP39 （Satoshi Labs Improvement Proposal）、ShamirのSecret Sharing Scheme（SSSS）を使用して秘密を部分に分割するスキームを区別して説明しています。
+
+- [ビットコインの難易度の粒度はどのようにエンコードされますか？]({{bse}} 92990) zndtoshiは、ヘッダーハッシュにゼロを追加すると難易度が指数関数的に増加する場合、マイニングの難易度をどのように細かく調整できるかを疑問に挙げています。Murchは`nBits`、*難易度*と*ターゲット*しきい値の関係、および[詳細な例と図][stack exchange harding target answer]へのリンクについて説明しています。
```

Comment:
@Ichiro0219 The second link in the stack exchange section is broken (it has an extra space in it). Can you correct?

Reply:
@bitschmidty Fixed. Please have a look.